### ISSUE TITLE
Updating a repository is never ending on frontend

### ISSUE THREAD

**Issue by zen2** [hacs/integration#1494]
## Installation details

| Description                | Value |
| -------------------------- | ----- |
| HACS version               | 1.5.2 |
| Home Assistant version     | 0.114.3 |
| Installation method for HA | Homeassistant repository overlay on gentoo linux |
| Web browser                | Firefox |
| Web browser version        | 79 / Linux |

## Checklist

- [X] I'm running the newest version of HACS <https://github.com/hacs/integration/releases/latest>
- [X] I have filled out the issue template to the best of my ability.
- [X] I have read <https://hacs.xyz/docs/issues>
- [X] This issue is related to the frontend of HACS.
- [X] This issue only contain 1 issue (if you have multiple issues, open one issue for each issue).

## Describe the issue

When I need to update a repository I click on his card then click update.
Then frontend show the progress indicator with "updating" text.
When update is finished (I check it at filesystem level ), the frontend stay in the same state with the progress indicator and "updating" text. That goes away only If I click outside the modal box.
I expect that once update is finished to to do not have anymore the progress indicator and updating text but a refresh of modal box with something telling that update process is finished.

### Steps to reproduce

1. Have a repository to update (1st screenshot)
2. Click on "update" to have modal box open (2nd screenshot)
3. Click on "update" to launch updating process and see circular indicator (3rd screenshot)
4. The frontend update process stay in this state (3rd screenshot) until I click outside the modal box

### Screenshots
![updating-1](https://user-images.githubusercontent.com/9433239/91448553-4d5fce80-e87a-11ea-9958-42ccf16f8fb1.png)
![updating-2](https://user-images.githubusercontent.com/9433239/91448561-50f35580-e87a-11ea-8cae-fcce3273c449.png)
![updating-3](https://user-images.githubusercontent.com/9433239/91448566-5355af80-e87a-11ea-9794-6e2d6d252e3a.png)

## Javascript logs from your browser console

<details>
  <summary>Logs</summary>

```text
 Uncaught (in promise) TypeError: this.status is undefined
    value http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:3898
    handleEvent http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    __boundHandleEvent http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    commit http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    update http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    __commitTemplateResult http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    commit http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    render http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    render http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    update http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    performUpdate http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    _enqueueUpdate http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    _requestUpdate http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    initialize http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    initialize http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    re http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    we http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    o http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:3625
    F http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:3855
    value http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:4543
    update http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    performUpdate http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    _enqueueUpdate http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    _requestUpdate http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    set http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    value http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:4580
    value http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:4543
    value http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:3090
    handleEvent http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
    __boundHandleEvent http://zendomo.local:8123/hacsfiles/frontend-202008231558.js:1
```

</details>

**Comment by ludeeus** [comment]
duplicate of https://github.com/hacs/integration/issues/1448
